### PR TITLE
Revert "Remove "intelligent" time scheduling instruction"

### PR DESCRIPTION
**Pull request by adamaltman**
Reverts Rebilly/api-definitions#585

### CODE REVIEW

**Review by ashkarpetin — DISMISSED** [review]
It never worked, so we removed it

**Comment by adamaltman** [comment]
Please see the issue comments though: https://github.com/Rebilly/backend/issues/13896

**Review by ashkarpetin — APPROVED** [review]
We should fix it instead of removing it

File: openapi/components/schemas/Scheduling/Methods/intelligent.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        type: integer
+        description: The number of the units.
+        minimum: 1
+      unit:
+        oneOf:
+          - $ref: ../../TimeUnit.yaml
+          - $ref: ../../TimePluralUnit.yaml
+    required:
+      - duration
+      - unit
```
Comment (ashkarpetin) [inline openapi/components/schemas/Scheduling/Methods/intelligent.yaml:17]: We need to check if these configuration parameters are valid for `intelligent` scheduling

**Review by viceromag — APPROVED** [review]

**Review by XaosSintez — APPROVED** [review]

**Pull request merged**